src/redux/catergories/catergories.js

```diff
@@ -0,0 +1,20 @@
+const checkcategory = 'bookstore/categories/checkcategory';
+
+const initialcategories = {
+  category: [],
+};
+
```

Comment:
- [ ] Great job working on this milestone. As per project requirements, you are supposed to make use of/define `slice` for the redux state. At the moment, you have not used slices. Even the import of `createSlice ` from `"@reduxjs/toolkit"` you don't have it.
You can have something similar to below snippet before the reducer for checking status
```javascript
import { createSlice } from "@reduxjs/toolkit"

export const categoriesSlice = createSlice({
  name: "categories",
  initialState: {
    categories: [],
  },
    .
    .
    .
```
![image](https://github.com/shihabrafio/BookStore/assets/62849872/fb7bec14-9ac8-43b5-b832-f17ce85ced9b)